### PR TITLE
docs: unpublishing this page, as it refers to labs

### PR DESCRIPTION
**Pull request by sharepointoscar**
This is not a supported scenario as far as I know.

### CODE REVIEW

**Comment by hervelemeur** [comment]
@sharepointoscar there are 2 methods in this page, one of them isn't concerned by jxl, shouldn't it be kept?

File: content/en/docs/reference/multi-cluster.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ lastmod: 2018-01-02
 categories: [getting started]
 keywords: [install,kubernetes]
 weight: 30
+draft: true
```
Comment (hferentschik) [inline content/en/docs/reference/multi-cluster.md:11]: I don't think the whole page should become a draft. It is just the `jxl` part which should be removed.

Unpublishing this page causes these link errors:

```
- ./tmp-website/community/office_hours/2019-04-19/index.html
  *  internally linking to /docs/reference/multi-cluster/, which does not exist (line 0)
     <a href="/docs/reference/multi-cluster/">multi cluster</a>
  *  internally linking to /docs/reference/multi-cluster/, which does not exist (line 0)
     <a href="/docs/reference/multi-cluster/">Multi-cluster</a>
htmlproofer 3.15.3 | Error:  HTML-Proofer found 2 failures!
```

Comment (ethangj) [reply content/en/docs/reference/multi-cluster.md:11]: That's fair. +1 on just removing the JXL bit. @sharepointoscar @vfarcic

Comment (sharepointoscar) [reply content/en/docs/reference/multi-cluster.md:11]: True true, let me edit that portion and publish an update!

**Comment by sharepointoscar** [comment]
> @sharepointoscar there are 2 methods in this page, one of them isn't concerned by jxl, shouldn't it be kept?

Absolutely correct!  I just edited the page and republishing shortly!  Thank you @hervelemeur

**Pull request merged**